thunder/tests/opinfos.py

```diff
@@ -5982,6 +5982,17 @@ def std_error_generator(op, device, dtype=torch.float32, **kwargs):
 reduction_ops.append(std_opinfo)
 
 
+std_mean_opinfo = OpInfo(
+    ltorch.std_mean,
+    supports_grad=True,
+    sample_input_generator=std_sample_generator,
+    error_input_generator=std_error_generator,
+    torch_reference=torch.std_mean,
+    dtypes=(datatypes.floating,),
```

Comment:
There are checks in the meta for complex types, but they are omitted from testing.  I know there are other issues with testing complex types, but were they left out here for a reason?